teamsyncd/teamsync.cpp

```diff
@@ -65,7 +66,7 @@ void TeamSync::addLag(const string &lagName, int ifindex, bool admin_state,
         return;
 
     /* Start track the team instance */
-    TeamPortSync *sync = new TeamPortSync(lagName, ifindex, &m_lagTable);
+    TeamPortSync *sync = new TeamPortSync(lagName, ifindex, &m_lagMemberTable);
     m_select->addSelectable(sync);
     m_teamPorts[lagName] = shared_ptr<TeamPortSync>(sync);
```

Comment:
i think the reason is that the original author of this code wanted to pass "Selectable *" to addSelectable() function as this function right now only supports weak_ptr.
anyway, this is a separate issue from this pull request.